faucet/privateKeysLoader.go

```diff
@@ -58,7 +58,7 @@ func (pkl *PrivateKeysLoader) PrivateKeysByShard() (map[uint32][]crypto.PrivateK
 	}
 
 	for _, privKeyBytes := range privKeysBytes {
-		privKeyBytes, err := hex.DecodeString(string(privKeyBytes))
+		privKeyBytes, err = hex.DecodeString(string(privKeyBytes))
```

Comment:
please rename `privKeyBytes` as it is the same with the var name in the for loop

Reply:
done.